docs/cli-reference/gardenadm/gardenadm_token.md

```diff
@@ -22,8 +22,8 @@ Manage bootstrap and discovery tokens for gardenadm join
 ### SEE ALSO
 
 * [gardenadm](gardenadm.md)	 - gardenadm bootstraps and manages autonomous shoot clusters in the Gardener project.
-* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the server
-* [gardenadm token delete](gardenadm_token_delete.md)	 - Delete a bootstrap token on the server
-* [gardenadm token generate](gardenadm_token_generate.md)	 - Generate a random bootstrap token
-* [gardenadm token list](gardenadm_token_list.md)	 - List all bootstrap tokens on the server
+* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the cluster for joining a node
```

Comment:
Can you extend the getting started docs?

Reply:
Doesn't it make more sense to adapt the doc with https://github.com/gardener/gardener/pull/11942 when there is actually use of `gardenadm token`? Without the `join` command, it's pretty useless to create a bootstrap token.

Reply:
Moved to https://github.com/gardener/gardener/pull/11942/files#r2075503093